101-sql-vm-new-storage-ultrassd/azuredeploy.parameters.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualMachineName": {
+      "value": "GEN-UNIQUE-10"
+    },
+    "adminUsername": {
+      "value": "GEN-UNIQUE-10"
+    },
+    "adminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "existingVirtualNetworkName": {
+      "value": "GET-PREREQ-existingVirtualNetworkName"
+    },
+    "existingSubnetName": {
+      "value": "GET-PREREQ-existingSubnetName"
+    },
+    "existingVnetResourceGroup": {
+      "value": "GET-PREREQ-existingVnetResourceGroup"
+    },
+    "location": {
+      "value": "East US 2"
```

Comment:
are there location restrictions for this workload?  If so, add allowedValues to the template if not, remove this

Reply:
Added allowedValues for location